lib/pbench/test/unit/server/query_apis/commons.py

```diff
@@ -300,7 +317,9 @@ def test_http_exception(
             pytest.skip("skipping " + self.test_http_exception.__name__)
 
         index = self.build_index(
-            server_config, self.date_range(self.payload["start"], self.payload["end"])
+            server_config,
+            provide_metadata,
```

Comment:
As I said above, I don't think you need to supply `provide_metadata` here as an argument.

Ditto line 354.

Reply:
I get that we need to specify a reference to the `provide_metadata` fixture in the function signature (e.g., at line 311), so that the fixture will be executed and set up the metadata, but I don't see how providing it here (at line 321) accomplishes anything.

What we require is a side-effect of the fixture's execution; the fixture doesn't yield any explicit value, so referencing it inside the function presumably just produces a `None` value (or whatever the last statement in the the `provide_metadata()` function evaluates to).